.github/actions/templates/avm-validateModuleDeployment/action.yml

```diff
@@ -419,6 +418,28 @@ runs:
 
           Write-Output '::endgroup::'
 
+    # [Azure login] task(s)
```

Comment:
Hey @sebassem thanks for this! I see you mention Pester test but I guess you are referring to the post-deployment ones? Historically we got issues with the removal part, exactly for the reason you mention (longer deployment) although it's not clear or fully predictable when the token expires.

As this is the shared action for deployment validation across the library, we'd need to test it on a representative number of modules before merging, and check if this also fixes the issue for the other exempted modules. Happy to support on that.

Reply:
@eriqua I have added a couple of tests above